README.md

```diff
@@ -32,6 +32,7 @@ or just [browse through the documentation](https://docs.fluxcd.io/projects/helm-
 As Flux Helm Operator is Open Source, integrations are very straight-forward. Here are
 a few popular ones you might want to check out:
 
+- [Progressive Delivery workshop (Helm v3)](https://helm.workshop.flagger.dev/)
```

Comment:
Maybe mention 'alpha' so people don't go looking for something that isn't officially there yet.